### ISSUE TITLE
Pretrained Model Details | Requesting Clarification

### ISSUE THREAD

**Issue by AmoghTiwari** [kasvii/PMCE#9]
Hello,

Thank you for your great work, and making the code publicly available.

In the [link](https://1drv.ms/f/s!Alg46BPWJg_XgXCTabSm_2wRfIN8?e=BnpbLV) provided to the pretrained models, there are multiple pretrained models available. While you mention that for demo  purposes, we should use the "mesh_vis.pth.tar" model, it is not clear what the other models are.

My understanding is that the other models are specific models for evaluation on different datasets. If my understanding is correct, then could you please clarify whether you perform individual training and testing for each dataset, or whether you train a single model on a mixture of the datasets and then perform all testing with that single model?

**Comment by kasvii** [comment]
You are correct. The models are trained and evaluated on different datasets to compare with other methods, as shown in Table 1 of the [paper](https://openaccess.thecvf.com/content/ICCV2023/papers/You_Co-Evolution_of_Pose_and_Mesh_for_3D_Human_Body_Estimation_ICCV_2023_paper.pdf). You can reproduce them under the training [instruction](https://github.com/kasvii/PMCE/tree/581d074c3142a22ddaf02f16ce1dc88fe94ea4e1?tab=readme-ov-file#train). For 3dpw, the model was trained on mixed 2D and 3D datasets, including 3dpw, h36m, mpii3d, coco, and mpii, and tested on 3dpw testing set. For mpii3d, we take 3dpw's checkpoint and directly test on mpii3d testing set. For h36m, the model was trained on h36m training set and tested on h36m testing set. 'mesh_vis.pth.tar' is one of 3dpw's checkpoints. Hope this reply helps you.

**Comment by AmoghTiwari** [comment]
Thank you for clarifying this in detail :). This makes it quite clear.
Closing the issue.

**Comment by AmoghTiwari** [comment]
As a follow up to my previous query, is this the commonly used strategy adopted by previous works as well? My understanding is that previous works like [MPS-Net ](https://mps-net.github.io/MPS-Net/) or [TCMR](https://github.com/hongsukchoi/TCMR_RELEASE) use a single pretrained checkpoint (trained on a combination of datasets) for evaluation across all the test datasets.

**Comment by kasvii** [comment]
MPS-Net and TCMR also use different settings for diverse test datasets, as indicated [here](https://github.com/hongsukchoi/TCMR_RELEASE/tree/master/configs). In contrast to MPS-Net and TCMR, PMCE introduces the pose branch, which is influenced by[ joint set types](https://github.com/hongsukchoi/Pose2Mesh_RELEASE/blob/master/asset/joint_sets.png). Thus, PMCE mainly follows the settings of the pose-based method, [Pose2Mesh](https://github.com/hongsukchoi/Pose2Mesh_RELEASE).

**Comment by Dragon2938734** [comment]
hello, the pretrained model you provide is 393M, but i reproduced following the training introduction is 1.1G, is there any problems cause this difference? Thank you and look forward to your reply!

**Comment by kasvii** [comment]
> hello, the pretrained model you provide is 393M, but i reproduced following the training introduction is 1.1G, is there any problems cause this difference? Thank you and look forward to your reply!

Hello, to save memory for easy downloading, the uploaded pretrained model only contains the model parameters. In contrast, the reproduced one contains additional optimizer parameters.
